include/mbedtls/pk.h

```diff
@@ -789,6 +797,43 @@ static inline mbedtls_ecp_keypair *mbedtls_pk_ec(const mbedtls_pk_context pk)
             return NULL;
     }
 }
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Do we need to make these functions public? I'd be inclined to keep internal, declared for example in `library/pk_internal.h` or `library/pk_psa.h`, so that other library modules can use them, but we're free to change them any time we want - as opposed to fully public functions that need to be kept unchanged until Mbed TLS 4.0.